### PR TITLE
[Chore]: Technical: Translate redusers (UI-state and provider-states)

### PR DESCRIPTION
**Pull request by HeimEndyd**

### CODE REVIEW

**Comment by HeimEndyd** [comment]
We are waiting for approval of this PR [https://github.com/keplergl/kepler.gl/pull/1725](url)

**Review by heshan0131 — APPROVED** [review]

**Pull request merged**
